README.md

```diff
@@ -1,10 +1,15 @@
 # Zest_Display_LCD
 
 Zest_Display_LCD board support for Zephyr OS.
-
 ## Usage
-This board enables the following sensors:
-- List Zest_Display_LCD sensors here
+This board enable the following driver:
+- [ILITEK ILI9163C](https://www.buydisplay.com/download/ic/ILI9163.pdf) display driver
+
+:bulb: Display driver should also be added to your workspace:
+- [ILITEK ILI9163C driver for Zephyr OS](https://github.com/catie-aq/zephyr_ilitek-ili9163c)
+
+The `zephyr,display` label is exposed by the overlay.
+Shield name: `zest_display_lcd`
 
-:bulb: Sensors' drivers should also be added to your workspace:
-- List Zephyr OS drivers here
+A full demonstration using the LVGL library is available here:
+[zephyr_zest-display-lcd_demo](https://github.com/catie-aq/zephyr_zest-display-lcd_demo).
```

Comment:
Please remove link to the demo, it will be included in 6TRON module samples.

See https://github.com/catie-aq/zephyr_6tron-manifest/issues/110